src/main/java/org/broadinstitute/hellbender/tools/exome/alleliccount/AllelicCountWithPhasePosteriors.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.hellbender.tools.exome.alleliccount;
+
+import org.broadinstitute.hellbender.tools.exome.allelefraction.AlleleFractionModeller;
+import org.broadinstitute.hellbender.tools.exome.alleliccount.AllelicCountTableColumn.AllelicCountTableVerbosity;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+/**
+ * Represents an {@link AllelicCount} with posterior probabilities for each site being ref minor, alt minor,
+ * or an outlier, according to a model fit by {@link AlleleFractionModeller}.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public final class AllelicCountWithPhasePosteriors extends AllelicCount {
+    private final double refMinorProb;
+    private final double altMinorProb;
+    private final double outlierProb;
+
+    /**
+     * Construct the {@link AllelicCountWithPhasePosteriors} object.  If unnormalized probabilities are passed,
+     * they will be normalized.
+     * @param count         {@link AllelicCount} of any verbosity (see {@link AllelicCountTableVerbosity})
+     * @param refMinorLogProb  log posterior probability of ref-minor phase (can be unnormalized)
+     * @param altMinorLogProb  log posterior probability of alt-minor phase (can be unnormalized)
+     * @param outlierLogProb   log posterior probability of outlier phase (can be unnormalized)
+     */
+    public AllelicCountWithPhasePosteriors(final AllelicCount count,
+                                           final double refMinorLogProb, final double altMinorLogProb, final double outlierLogProb) {
+        super(Utils.nonNull(count, "AllelicCount cannot be null."));
+        ParamUtils.isFinite(refMinorLogProb, "Cannot construct AllelicCountWithPhasePosteriors with non-finite ref-minor probability at: " + count.getInterval());
```

Comment:
i'll leave it for now and will try to avoid it in the future.  thanks in advance for cleaning things up!